website_snippet_product_category/controllers/website.py

```diff
@@ -13,12 +13,14 @@ class Website(http.Controller):
         auth="public",
         website=True,
     )
-    def render_product_category(self, template, **kwargs):
+    def render_product_category(self, **kwargs):
```

Comment:
This will break downstream extensions.

I think what we need here is a whitelist of allowed templates, and a 403 if the template isn't in the list.